examples/triangle.py

```diff
@@ -47,90 +47,92 @@ def fragment_shader(
 def main(canvas):
     """ Regular function to setup a viz on the given canvas.
     """
-    adapter = wgpu.requestAdapter(powerPreference="high-performance")
-    device = adapter.requestDevice(extensions=[], limits=wgpu.GPULimits())
+    adapter = wgpu.request_adapter(power_preference="high-performance")
+    device = adapter.request_device(extensions=[], limits={})
     return _main(canvas, device)
 
 
-async def mainAsync(canvas):
+async def main_async(canvas):
     """ Async function to setup a viz on the given canvas.
     """
-    adapter = await wgpu.requestAdapterAsync(powerPreference="high-performance")
-    device = await adapter.requestDeviceAsync(extensions=[], limits=wgpu.GPULimits())
+    adapter = await wgpu.request_adapter_async(power_preference="high-performance")
+    device = await adapter.request_device_async(extensions=[], limits={})
     return _main(canvas, device)
 
 
 def _main(canvas, device):
 
-    vshader = device.createShaderModule(code=vertex_shader)
-    fshader = device.createShaderModule(code=fragment_shader)
+    vshader = device.create_shader_module(code=vertex_shader)
+    fshader = device.create_shader_module(code=fragment_shader)
 
-    bind_group_layout = device.createBindGroupLayout(bindings=[])  # zero bindings
-    bind_group = device.createBindGroup(layout=bind_group_layout, bindings=[])
+    bind_group_layout = device.create_bind_group_layout(bindings=[])  # zero bindings
+    bind_group = device.create_bind_group(layout=bind_group_layout, bindings=[])
 
-    pipeline_layout = device.createPipelineLayout(bindGroupLayouts=[bind_group_layout])
+    pipeline_layout = device.create_pipeline_layout(
+        bind_group_layouts=[bind_group_layout]
+    )
 
-    render_pipeline = device.createRenderPipeline(
+    render_pipeline = device.create_render_pipeline(
         layout=pipeline_layout,
-        vertexStage={"module": vshader, "entryPoint": "main"},
-        fragmentStage={"module": fshader, "entryPoint": "main"},
-        primitiveTopology=wgpu.PrimitiveTopology.triangle_list,
-        rasterizationState={
-            "frontFace": wgpu.FrontFace.ccw,
-            "cullMode": wgpu.CullMode.none,
-            "depthBias": 0,
-            "depthBiasSlopeScale": 0.0,
-            "depthBiasClamp": 0.0,
+        vertex_stage={"module": vshader, "entry_point": "main"},
+        fragment_stage={"module": fshader, "entry_point": "main"},
+        primitive_topology=wgpu.PrimitiveTopology.triangle_list,
+        rasterization_state={
+            "front_face": wgpu.FrontFace.ccw,
+            "cull_mode": wgpu.CullMode.none,
+            "depth_bias": 0,
+            "depth_bias_slope_scale": 0.0,
+            "depth_bias_clamp": 0.0,
         },
```

Comment:
Perhaps this illustrates the changes best: 
* the function name is snake_case
* the direct arguments are snake_case
* the "sub-arguments" (keys of args that are dicts) are snake_case
* enums and flags are unchanged.